modular-docs-manual/content/topics/module_guidelines-concept.adoc

```diff
@@ -33,6 +33,37 @@ Do not include any instructions to perform an action, such as executing a comman
 Action items belong in procedure modules.
 See also link:http://www.informationmapping.com/fspro2013-tutorial/infotypes/infotype2.html[The Six Information Types] at _informationmapping.com_ for ways to present different types of conceptual information: principle, concept, structure, process, fact.
 
+If the concept module is large and complex, either split the concept module into multiple standalone concept modules or use subheadings in the module to structure the content for improved reader navigation. If you use subheadings in a concept module, you can use the `[discrete]` tag to exclude the heading from the table of contents, if needed. However, in many cases, subheadings are helpful to include in the table of contents.
```

Comment:
@sterobin I'm thinking about something along these lines (not this wording, I'm just throwing my thoughts out here):

If you are writing a concept module that has a lot of content, you might decide to break that content into separate modules. This approach is good if modules can standalone, meaningful units of information. Creating separate modules lets you easily move content around and re-use it. It's also easier to update, edit, and delete content that is broken into separate modules. 

However, as an alternative to creating separate modules, you can add subheadings to nest content that is directly, and uniquely, related to the concept you're writing about. Subheadings are a good way to organize large concepts into sections that improve reader navigation. You can also add the `[discrete]` attribute to subheadings so that they do not appear in the table of content. However, doing this might reduce retrieve-ability as the headings do not display in the high-level TOC so users can't find floating sections when scanning the document.

Reply:
Thanks @oraNod , yeah makes sense. I'll see how I can tweak, a bit later.

Reply:
@oraNod I reworded a bit to lean more toward modularizing, or as an alternative, subdividing. I've intentionally kept it fairly simple though. I don't know that we want to get into more detail or be any more forceful than than about splitting into modules. If anything, the point of this PR is to encourage writers to consider subheadings more often so that they'll stop over-splitting concept modules into artificial mini-modules, which is what I'm observing. So I've proposed the rewording as a compromise. Let me know what you think.